backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_OrderLine.java

```diff
@@ -1159,6 +1159,27 @@ public interface I_C_OrderLine
 	ModelColumn<I_C_OrderLine, Object> COLUMN_IsActive = new ModelColumn<>(I_C_OrderLine.class, "IsActive", null);
 	String COLUMNNAME_IsActive = "IsActive";
 
+	/**
+	 * Set Getrennt fakturieren.
```

Comment:
any idea why this is german?
does `IsAllowSeparateInvoicing` element have a proper en_US trl?

Reply:
Think I've added the en_US trl after generating the models.